BSP/Inc/BSP_Timer.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Header file for the library to interact
+ * with the timers
+ */
+#include "common.h"
+#include <bsp.h>
+
+typedef enum {Time1=0, Time2=1} Time_t;

```

Comment:
See style guide for the naming conventions/capitalization of enums

Reply:
Also, you don't need to specify every value. Once you specify that the first value is 0, the rest will increment from there.